java/java.j2seplatform/build.xml

```diff
@@ -25,7 +25,7 @@
 
     <target name="probe" depends="init">
         <mkdir dir="build/probe/classes"/>
-        <javac srcdir="probesrc" destdir="build/probe/classes" source="1.3" debug="true" deprecation="true" target="1.1"/>
+        <javac srcdir="probesrc" destdir="build/probe/classes" source="${javac.source}" debug="true" deprecation="true" target="${javac.target}"/>
```

Comment:
This, for example, seems to have broken the probing functionality for JDK 7 and below, and unnecessarily: you could have just gone down to Java 6, breaking it only for JDK 5 and below.

Reply:
thanks for providing the fix
https://github.com/apache/netbeans/pull/1281